lib/std/compress.zig

```diff
@@ -1,13 +1,21 @@
 const std = @import("std.zig");
 
-pub const deflate = @import("compress/deflate.zig");
-pub const gzip = @import("compress/gzip.zig");
 pub const lzma = @import("compress/lzma.zig");
 pub const lzma2 = @import("compress/lzma2.zig");
 pub const xz = @import("compress/xz.zig");
-pub const zlib = @import("compress/zlib.zig");
 pub const zstd = @import("compress/zstandard.zig");
 
+pub const flate = @import("compress/flate/root.zig").flate;
+pub const gzip = @import("compress/flate/root.zig").gzip;
+pub const zlib = @import("compress/flate/root.zig").zlib;
+
+// Version 1 interface
+pub const v1 = struct {
+    pub const deflate = @import("compress/deflate.zig");
+    pub const gzip = @import("compress/gzip.zig");
+    pub const zlib = @import("compress/zlib.zig");
+};
+
```

Comment:
This was very considerate of you to do, however, please go ahead and delete the old code. At this point in time in the zig project, it's the best path forward.

I cannot wait until I don't have to see "Go non-regression test..." flash in front of my eyes every time I run the std lib tests :slightly_smiling_face:

Reply:
Exactly what I was expecting from you ;-)